modules/rest-api/pages/rest-auditing.adoc

```diff
@@ -23,7 +25,8 @@ This includes the enablement and disablement of auditing, the establishment of a
 The `GET` method used with the `/settings/audit/descriptors` URI returns a document that lists _filterable_ audit events; providing an _id_, _name_, _module_, and _description_ for each.
 The _id_ can be used in interpreting the output from `POST /settings/audit`, and in determining input-values for `GET /settings/audit`.
 A _filterable_ event is an event that can be individually disabled, even when event-auditing for the node is enabled.
-Events that are not filterable are not included in the list returned by `GET /settings/audit/descriptors`.
+Events that are not filterable are not included in the list returned by `GET /settings/audit/descriptors`. +
+Events that are not filterable can be retrieved using the `GET` method /settings/audit/nonFilterableDescriptors
```

Comment:
Should the path be in backticks?

Events that are not filterable can be retrieved using the `GET` method `/settings/audit/nonFilterableDescriptors`.